### PR TITLE
Azure Monitor: Docs update for multiple resources

### LINKED ISSUE #62331: Documentation
Documentation updates for the multiple resource picker

### PR DESCRIPTION
**Pull request by andresmgot**
**What is this feature?**

It is now possible to select multiple resource groups and names in a single Azure Monitor query.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/62331

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/datasources/azure-monitor/query-editor/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ In contrast, Azure Monitor Logs can store a variety of data types, each with the
 
 1. In a Grafana panel, select the **Azure Monitor** data source.
 1. Select the **Metrics** service.
-1. Select a resource from which to metrics by using the subscription, resource group, resource type, and resource fields.
+1. Select a resource from which to metrics by using the subscription, resource group, resource type, and resource fields. Multiple resources can also ben selected as long as they belong to the same subscription, region and resource type. Note that only a limited amount of resource types support this feature.
```
Comment (aangelisc) [inline docs/sources/datasources/azure-monitor/query-editor/index.md:47]:
```suggestion
1. Select a resource from which to query metrics by using the subscription, resource group, resource type, and resource fields. Multiple resources can also be selected as long as they belong to the same subscription, region and resource type. Note that only a limited amount of resource types support this feature.
```

Comment (aangelisc) [reply docs/sources/datasources/azure-monitor/query-editor/index.md:47]: Maybe we should reference the list of supported resource types?

File: docs/sources/datasources/azure-monitor/query-editor/index.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ You can also perform complex analysis of Logs data by using KQL.
 
 1. In a Grafana panel, select the **Azure Monitor** data source.
 1. Select the **Logs** service.
-1. Select a resource to query.
+1. Select a resource to query. Multiple resources can be selected as far as they are of the same type.
```
Comment (aangelisc) [inline docs/sources/datasources/azure-monitor/query-editor/index.md:113]:
```suggestion
1. Select a resource to query. Multiple resources can be selected as long as they are of the same type.
```

File: docs/sources/datasources/azure-monitor/template-variables/index.md [hunk 3]
```diff
@@ -32,16 +32,16 @@ For an introduction to templating and template variables, refer to the [Templati
...
+| **Subscriptions**   | Returns subscriptions.                                                                                             |
+| **Resource Groups** | Returns resource groups for a specified. Supports multi-value. subscription.                                       |
+| **Namespaces**      | Returns metric namespaces for the specified subscription and resource group.                                       |
+| **Resource Names**  | Returns a list of resource names for a specified subscription, resource group and namespace. Supports multi-value. |
+| **Metric Names**    | Returns a list of metric names for a resource.                                                                     |
+| **Workspaces**      | Returns a list of workspaces for the specified subscription.                                                       |
+| **Logs**            | Use a KQL query to return values.                                                                                  |
+| **Resource Graph**  | Use an ARG query to return values.                                                                                 |
```
Comment (aangelisc) [inline docs/sources/datasources/azure-monitor/template-variables/index.md:44]: May be worth adding a reference to the new `Region` variable editor

File: docs/sources/datasources/azure-monitor/template-variables/index.md [hunk 4]
```diff
@@ -65,3 +65,13 @@ Perf
 | summarize avg(CounterValue) by bin(TimeGenerated, $__interval), Computer
 | order by TimeGenerated asc
 ```
+
+### Multi-value variables
+
+It is possible to select multiple values for **Resource Groups** and **Resource Names** and use a single metrics query pointing to those values as far as they:
```
Comment (aangelisc) [inline docs/sources/datasources/azure-monitor/template-variables/index.md:71]:
```suggestion
It is possible to select multiple values for **Resource Groups** and **Resource Names** and use a single metrics query pointing to those values as long as they:
```

File: docs/sources/datasources/azure-monitor/template-variables/index.md [hunk 5]
```diff
@@ -65,3 +65,13 @@ Perf
...
+
+- Belong to the same subscription.
+- Are in the same region.
+- Are of the same type (namespace).
+
+Also, note that if a template variable pointing to multiple resource groups or names is used in other template variable as a parameter (e.g. to retrieve metric names), only the first value will be used. This means that the combination of the first resource group and name selected should be valid.
```
Comment (aangelisc) [inline docs/sources/datasources/azure-monitor/template-variables/index.md:77]:
```suggestion
Also, note that if a template variable pointing to multiple resource groups or names is used in another template variable as a parameter (e.g. to retrieve metric names), only the first value will be used. This means that the combination of the first resource group and name selected should be valid.
```

**Review by aangelisc — APPROVED** [review]
All good! Just a couple typos 😊

File: docs/sources/datasources/azure-monitor/query-editor/index.md (hunk 1 above)
Comment (andresmgot) [reply docs/sources/datasources/azure-monitor/query-editor/index.md:47]: There is no good point to reference, pointing code seems too verbose and there is no docs related to this (I mean Azure docs).

File: docs/sources/datasources/azure-monitor/template-variables/index.md (hunk 3 above)
Comment (andresmgot) [reply docs/sources/datasources/azure-monitor/template-variables/index.md:44]: right! I forgot :)

**Comment by kayodeprinceMS** [comment]
Thanks for the docs updates! Azure docs mentioning supported resource types listed[ here](https://learn.microsoft.com/azure/azure-monitor/essentials/metrics-dynamic-scope#select-multiple-resources).

**Comment by aangelisc** [comment]
Hey @kayodeprinceMS, is there a static reference for that image that we could use for reference?

**Comment by kayodeprinceMS** [comment]
Not that I know. Will make the ask to Azure docs.

**Pull request merged**